iree/hal/vulkan/vulkan_device.cc

```diff
@@ -38,6 +38,7 @@
 #include "iree/hal/vulkan/pipeline_executable_layout.h"
 #include "iree/hal/vulkan/status_util.h"
 #include "iree/hal/vulkan/vma_allocator.h"
+#include "iree/hal/vulkan/vulkan_driver.h"
```

Comment:
This is missing a build dependency (which looks like it would be a circular dependency), which is causing a build failure with Bazel:

```
ERROR: /tmpfs/src/github/iree/iree/hal/vulkan/BUILD:375:1: Couldn't build file iree/hal/vulkan/_objs/vulkan_device/vulkan_device.pic.o: C++ compilation of rule '//iree/hal/vulkan:vulkan_device' failed (Exit 1)
iree/hal/vulkan/vulkan_device.cc:41:10: fatal error: 'iree/hal/vulkan/vulkan_driver.h' file not found
#include "iree/hal/vulkan/vulkan_driver.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
yeah, just try passing in the VkInstance directly instead of casting it out to avoid that cycle

Reply:
... shakes fist at overly fine-grained build rules...

fixed